src/main/kotlin/com/klaxit/hiddensecrets/HiddenSecretsPlugin.kt

```diff
@@ -24,22 +25,28 @@ open class HiddenSecretsPlugin : Plugin<Project> {
         private const val KOTLIN_FILE_NAME = "Secrets.kt"
 
         //Tasks
+        const val TASK_GROUP = "Hide secrets"
         const val TASK_UNZIP_HIDDEN_SECRETS = "unzipHiddenSecrets"
         const val TASK_COPY_CPP = "copyCpp"
         const val TASK_COPY_KOTLIN = "copyKotlin"
         const val TASK_HIDE_SECRET = "hideSecret"
+        const val TASK_HIDE_SECRET_FROM_PROPS = "hideSecretFromProps"
```

Comment:
I prefer the project code to be super clear, so I suggest to use instead :
` const val TASK_HIDE_SECRETS_FROM_PROPERTIES = "hideSecretsFromProperties"`